CMakeLists.txt

```diff
@@ -81,10 +81,17 @@ set(_alpaka_INCLUDE_DIRECTORY "${_alpaka_ROOT_DIR}/include")
 set(_alpaka_SUFFIXED_INCLUDE_DIR "${_alpaka_INCLUDE_DIRECTORY}/alpaka")
 
 # the sequential accelerator is required for the tests and examples
-if(NOT DEFINED alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE AND (alpaka_BUILD_EXAMPLES OR BUILD_TESTING))
-  option(alpaka_ACC_CPU_B_SEQ_T_SEQ_ENABLE "enable alpaka serial accelerator" ON)
+if(alpaka_BUILD_EXAMPLES OR BUILD_TESTING)
+  if (NOT alpaka_ACC_GPU_CUDA_ONLY_MODE OR alpaka_ACC_GPU_HIP_ONLY_MODE)
```

Comment:
I am not 100% sure but I would say this line should be

```
if (NOT (alpaka_ACC_GPU_CUDA_ONLY_MODE OR alpaka_ACC_GPU_HIP_ONLY_MODE) )
```

In your current version CMake would enable the serial accelerator if `alpaka_ACC_GPU_HIP_ONLY_MODE` is true because as I know OR has not have a higher priority than NOT.
Please correct me if I am wrong!

Reply:
Thanks - I confirmed locally that you're correct, and either way I always prefer brackets over remembering operator precedence.